Makefile.docker

```diff
@@ -48,8 +48,9 @@ image-controller: base-image-controller
 	cp ./pkg/${CONTROLLER}/Dockerfile ./build/${CONTROLLER}/
 	docker build --network host --force-rm --tag ${REGISTRY}/${PROJECT}-${CONTROLLER}:${TAG} ./build/${CONTROLLER}
 
-image-extender:
+image-extender: download-grpc-health-probe
```

Comment:
I won't be able to build my project without access to the internet with this dependency.

Reply:
updated